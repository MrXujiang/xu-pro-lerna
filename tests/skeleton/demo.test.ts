import demoTest from '../demo';

demoTest('skeleton');
