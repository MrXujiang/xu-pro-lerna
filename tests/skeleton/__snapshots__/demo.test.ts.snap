// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`skeleton demos ðŸ“¸ renders ./packages/skeleton/src/demos/descriptions.tsx correctly 1`] = `
<div
  style="background: rgb(250, 250, 250); padding: 24px;"
>
  <div
    style="width: 100%;"
  >
    <div
      style="margin-bottom: 16px;"
    >
      <div
        class="ant-skeleton"
      >
        <div
          class="ant-skeleton-content"
        >
          <h3
            class="ant-skeleton-title"
            style="width: 185px;"
          />
        </div>
      </div>
      <div
        class="ant-skeleton ant-skeleton-element ant-skeleton-active"
      >
        <span
          class="ant-skeleton-button ant-skeleton-button-sm"
        />
      </div>
    </div>
    <div
      class="ant-card"
      style="border-top-right-radius: 0; border-top-left-radius: 0;"
    >
      <div
        class="ant-card-body"
      >
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-button ant-skeleton-button-sm"
            style="width: 100px; margin-bottom: 16px;"
          />
        </div>
        <div
          style="width: 100%; justify-content: space-between; display: flex;"
        >
          <div
            style="flex: 1; padding-left: 0px; padding-right: 24px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="margin-top: 0px;"
                />
              </div>
            </div>
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="margin-top: 8px;"
                />
              </div>
            </div>
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="margin-top: 8px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 24px; padding-right: 24px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="margin-top: 0px;"
                />
              </div>
            </div>
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="margin-top: 8px;"
                />
              </div>
            </div>
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="margin-top: 8px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 24px; padding-right: 0px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="margin-top: 0px;"
                />
              </div>
            </div>
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="margin-top: 8px;"
                />
              </div>
            </div>
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="margin-top: 8px;"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          style="margin-top: 32px;"
        >
          <div
            class="ant-skeleton ant-skeleton-element ant-skeleton-active"
          >
            <span
              class="ant-skeleton-button ant-skeleton-button-sm"
              style="width: 100px; margin-bottom: 16px;"
            />
          </div>
          <div
            style="width: 100%; justify-content: space-between; display: flex;"
          >
            <div
              style="flex: 1; margin-right: 24px; max-width: 300px;"
            >
              <div
                class="ant-skeleton ant-skeleton-active"
              >
                <div
                  class="ant-skeleton-content"
                >
                  <h3
                    class="ant-skeleton-title"
                    style="margin-top: 0px;"
                  />
                </div>
              </div>
              <div
                class="ant-skeleton ant-skeleton-active"
              >
                <div
                  class="ant-skeleton-content"
                >
                  <h3
                    class="ant-skeleton-title"
                    style="margin-top: 8px;"
                  />
                </div>
              </div>
              <div
                class="ant-skeleton ant-skeleton-active"
              >
                <div
                  class="ant-skeleton-content"
                >
                  <h3
                    class="ant-skeleton-title"
                    style="margin-top: 8px;"
                  />
                </div>
              </div>
            </div>
            <div
              style="flex: 1; align-items: center; justify-content: center;"
            >
              <div
                style="max-width: 300px; margin: auto;"
              >
                <div
                  class="ant-skeleton ant-skeleton-active"
                >
                  <div
                    class="ant-skeleton-content"
                  >
                    <h3
                      class="ant-skeleton-title"
                      style="margin-top: 0px;"
                    />
                  </div>
                </div>
                <div
                  class="ant-skeleton ant-skeleton-active"
                >
                  <div
                    class="ant-skeleton-content"
                  >
                    <h3
                      class="ant-skeleton-title"
                      style="margin-top: 8px;"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      style="padding: 0px 24px;"
    >
      <div
        class="ant-divider ant-divider-horizontal"
        role="separator"
        style="margin: 0px;"
      />
    </div>
    <div
      class="ant-card"
    >
      <div
        class="ant-card-body"
      >
        <div
          class="ant-skeleton ant-skeleton-element ant-skeleton-active"
        >
          <span
            class="ant-skeleton-button ant-skeleton-button-sm"
            style="width: 100px; margin-bottom: 16px;"
          />
        </div>
        <div
          style="display: flex; background: rgba(0, 0, 0, 0.02); padding: 24px 8px;"
        >
          <div
            style="flex: 1; padding-left: 0px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 75px; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 75px; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 75px; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 3; padding-left: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 75px; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          style="padding: 0px 0px;"
        >
          <div
            class="ant-divider ant-divider-horizontal"
            role="separator"
            style="margin: 0px;"
          />
        </div>
        <div
          style="display: flex; background: none; padding: 24px 8px;"
        >
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 3; padding-left: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          style="padding: 0px 0px;"
        >
          <div
            class="ant-divider ant-divider-horizontal"
            role="separator"
            style="margin: 0px;"
          />
        </div>
        <div
          style="display: flex; background: none; padding: 24px 8px;"
        >
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 3; padding-left: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          style="padding: 0px 0px;"
        >
          <div
            class="ant-divider ant-divider-horizontal"
            role="separator"
            style="margin: 0px;"
          />
        </div>
        <div
          style="display: flex; background: none; padding: 24px 8px;"
        >
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 3; padding-left: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          style="padding: 0px 0px;"
        >
          <div
            class="ant-divider ant-divider-horizontal"
            role="separator"
            style="margin: 0px;"
          />
        </div>
        <div
          style="display: flex; background: none; padding: 24px 8px;"
        >
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 1; padding-left: 20px; padding-right: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
          <div
            style="flex: 3; padding-left: 32px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100%; margin: 0px; height: 24px;"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          style="padding: 0px 0px;"
        >
          <div
            class="ant-divider ant-divider-horizontal"
            role="separator"
            style="margin: 0px;"
          />
        </div>
        <div
          style="display: flex; justify-content: flex-end; padding-top: 16px;"
        >
          <div
            class="ant-skeleton ant-skeleton-active"
          >
            <div
              class="ant-skeleton-content"
            >
              <h3
                class="ant-skeleton-title"
                style="margin: 0px; height: 32px; float: right; max-width: 630px;"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`skeleton demos ðŸ“¸ renders ./packages/skeleton/src/demos/list.tsx correctly 1`] = `
<div
  style="background: rgb(250, 250, 250); padding: 24px;"
>
  <div
    style="width: 100%;"
  >
    <div
      style="margin-bottom: 16px;"
    >
      <div
        class="ant-skeleton"
      >
        <div
          class="ant-skeleton-content"
        >
          <h3
            class="ant-skeleton-title"
            style="width: 185px;"
          />
        </div>
      </div>
      <div
        class="ant-skeleton ant-skeleton-element ant-skeleton-active"
      >
        <span
          class="ant-skeleton-button ant-skeleton-button-sm"
        />
      </div>
    </div>
    <div
      class="ant-card"
      style="margin-bottom: 16px;"
    >
      <div
        class="ant-card-body"
      >
        <div
          style="width: 100%; justify-content: space-between; display: flex;"
        >
          <div
            style="padding-left: 0px; flex: 1; margin-right: 16px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100px; margin-top: 0px;"
                />
              </div>
            </div>
            <div
              class="ant-skeleton ant-skeleton-element ant-skeleton-active"
            >
              <span
                class="ant-skeleton-button"
                style="height: 48px;"
              />
            </div>
          </div>
          <div
            style="padding-left: 42px; flex: 1; margin-right: 0px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100px; margin-top: 0px;"
                />
              </div>
            </div>
            <div
              class="ant-skeleton ant-skeleton-element ant-skeleton-active"
            >
              <span
                class="ant-skeleton-button"
                style="height: 48px;"
              />
            </div>
          </div>
          <div
            style="padding-left: 42px; flex: 1; margin-right: 0px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100px; margin-top: 0px;"
                />
              </div>
            </div>
            <div
              class="ant-skeleton ant-skeleton-element ant-skeleton-active"
            >
              <span
                class="ant-skeleton-button"
                style="height: 48px;"
              />
            </div>
          </div>
          <div
            style="padding-left: 42px; flex: 1; margin-right: 0px;"
          >
            <div
              class="ant-skeleton ant-skeleton-active"
            >
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width: 100px; margin-top: 0px;"
                />
              </div>
            </div>
            <div
              class="ant-skeleton ant-skeleton-element ant-skeleton-active"
            >
              <span
                class="ant-skeleton-button"
                style="height: 48px;"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-card"
    >
      <div
        class="ant-card-body"
        style="padding: 0px;"
      >
        <div
          class="ant-card"
          style="border-bottom-right-radius: 0; border-bottom-left-radius: 0;"
        >
          <div
            class="ant-card-body"
            style="padding-bottom: 8px;"
          >
            <div
              class="ant-space ant-space-horizontal ant-space-align-center"
              style="width: 100%; justify-content: space-between;"
            >
              <div
                class="ant-space-item"
                style="margin-right: 8px;"
              >
                <div
                  class="ant-skeleton ant-skeleton-element ant-skeleton-active"
                >
                  <span
                    class="ant-skeleton-button ant-skeleton-button-sm"
                    style="width: 200px;"
                  />
                </div>
              </div>
              <div
                class="ant-space-item"
              >
                <div
                  class="ant-space ant-space-horizontal ant-space-align-center"
                >
                  <div
                    class="ant-space-item"
                    style="margin-right: 8px;"
                  >
                    <div
                      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
                    >
                      <span
                        class="ant-skeleton-button ant-skeleton-button-sm"
                        style="width: 120px;"
                      />
                    </div>
                  </div>
                  <div
                    class="ant-space-item"
                  >
                    <div
                      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
                    >
                      <span
                        class="ant-skeleton-button ant-skeleton-button-sm"
                        style="width: 80px;"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-card"
        >
          <div
            class="ant-card-body"
            style="padding: 0px;"
          >
            <div
              class="ant-card"
              style="border-radius: 0;"
            >
              <div
                class="ant-card-body"
                style="padding: 24px;"
              >
                <div
                  style="width: 100%; display: flex; align-items: center; justify-content: space-between;"
                >
                  <div
                    style="max-width: 100%; flex: 1;"
                  >
                    <div
                      class="ant-skeleton ant-skeleton-active"
                    >
                      <div
                        class="ant-skeleton-content"
                      >
                        <h3
                          class="ant-skeleton-title"
                          style="width: 100px; margin-top: 0px;"
                        />
                        <ul
                          class="ant-skeleton-paragraph"
                          style="margin: 0px;"
                        >
                          <li
                            style="width: 61%;"
                          />
                        </ul>
                      </div>
                    </div>
                  </div>
                  <div
                    class="ant-skeleton ant-skeleton-element ant-skeleton-active"
                  >
                    <span
                      class="ant-skeleton-button ant-skeleton-button-sm"
                      style="width: 165px; margin-top: 12px;"
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              style="padding: 0px 24px;"
            >
              <div
                class="ant-divider ant-divider-horizontal"
                role="separator"
                style="margin: 0px;"
              />
            </div>
            <div
              class="ant-card"
              style="border-radius: 0;"
            >
              <div
                class="ant-card-body"
                style="padding: 24px;"
              >
                <div
                  style="width: 100%; display: flex; align-items: center; justify-content: space-between;"
                >
                  <div
                    style="max-width: 100%; flex: 1;"
                  >
                    <div
                      class="ant-skeleton ant-skeleton-active"
                    >
                      <div
                        class="ant-skeleton-content"
                      >
                        <h3
                          class="ant-skeleton-title"
                          style="width: 100px; margin-top: 0px;"
                        />
                        <ul
                          class="ant-skeleton-paragraph"
                          style="margin: 0px;"
                        >
                          <li
                            style="width: 61%;"
                          />
                        </ul>
                      </div>
                    </div>
                  </div>
                  <div
                    class="ant-skeleton ant-skeleton-element ant-skeleton-active"
                  >
                    <span
                      class="ant-skeleton-button ant-skeleton-button-sm"
                      style="width: 165px; margin-top: 12px;"
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              style="padding: 0px 24px;"
            >
              <div
                class="ant-divider ant-divider-horizontal"
                role="separator"
                style="margin: 0px;"
              />
            </div>
            <div
              class="ant-card"
              style="border-radius: 0;"
            >
              <div
                class="ant-card-body"
                style="padding: 24px;"
              >
                <div
                  style="width: 100%; display: flex; align-items: center; justify-content: space-between;"
                >
                  <div
                    style="max-width: 100%; flex: 1;"
                  >
                    <div
                      class="ant-skeleton ant-skeleton-active"
                    >
                      <div
                        class="ant-skeleton-content"
                      >
                        <h3
                          class="ant-skeleton-title"
                          style="width: 100px; margin-top: 0px;"
                        />
                        <ul
                          class="ant-skeleton-paragraph"
                          style="margin: 0px;"
                        >
                          <li
                            style="width: 61%;"
                          />
                        </ul>
                      </div>
                    </div>
                  </div>
                  <div
                    class="ant-skeleton ant-skeleton-element ant-skeleton-active"
                  >
                    <span
                      class="ant-skeleton-button ant-skeleton-button-sm"
                      style="width: 165px; margin-top: 12px;"
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              style="padding: 0px 24px;"
            >
              <div
                class="ant-divider ant-divider-horizontal"
                role="separator"
                style="margin: 0px;"
              />
            </div>
            <div
              class="ant-card"
              style="border-radius: 0;"
            >
              <div
                class="ant-card-body"
                style="padding: 24px;"
              >
                <div
                  style="width: 100%; display: flex; align-items: center; justify-content: space-between;"
                >
                  <div
                    style="max-width: 100%; flex: 1;"
                  >
                    <div
                      class="ant-skeleton ant-skeleton-active"
                    >
                      <div
                        class="ant-skeleton-content"
                      >
                        <h3
                          class="ant-skeleton-title"
                          style="width: 100px; margin-top: 0px;"
                        />
                        <ul
                          class="ant-skeleton-paragraph"
                          style="margin: 0px;"
                        >
                          <li
                            style="width: 61%;"
                          />
                        </ul>
                      </div>
                    </div>
                  </div>
                  <div
                    class="ant-skeleton ant-skeleton-element ant-skeleton-active"
                  >
                    <span
                      class="ant-skeleton-button ant-skeleton-button-sm"
                      style="width: 165px; margin-top: 12px;"
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              style="padding: 0px 24px;"
            >
              <div
                class="ant-divider ant-divider-horizontal"
                role="separator"
                style="margin: 0px;"
              />
            </div>
            <div
              class="ant-card"
              style="border-radius: 0;"
            >
              <div
                class="ant-card-body"
                style="padding: 24px;"
              >
                <div
                  style="width: 100%; display: flex; align-items: center; justify-content: space-between;"
                >
                  <div
                    style="max-width: 100%; flex: 1;"
                  >
                    <div
                      class="ant-skeleton ant-skeleton-active"
                    >
                      <div
                        class="ant-skeleton-content"
                      >
                        <h3
                          class="ant-skeleton-title"
                          style="width: 100px; margin-top: 0px;"
                        />
                        <ul
                          class="ant-skeleton-paragraph"
                          style="margin: 0px;"
                        >
                          <li
                            style="width: 61%;"
                          />
                        </ul>
                      </div>
                    </div>
                  </div>
                  <div
                    class="ant-skeleton ant-skeleton-element ant-skeleton-active"
                  >
                    <span
                      class="ant-skeleton-button ant-skeleton-button-sm"
                      style="width: 165px; margin-top: 12px;"
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              style="padding: 0px 24px;"
            >
              <div
                class="ant-divider ant-divider-horizontal"
                role="separator"
                style="margin: 0px;"
              />
            </div>
            <div
              class="ant-card"
              style="border-top-right-radius: 0; border-top-left-radius: 0;"
            >
              <div
                class="ant-card-body"
                style="display: flex; align-items: center; justify-content: center;"
              >
                <div
                  class="ant-skeleton ant-skeleton-element ant-skeleton-active"
                >
                  <span
                    class="ant-skeleton-button ant-skeleton-button-sm"
                    style="width: 102px;"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`skeleton demos ðŸ“¸ renders ./packages/skeleton/src/demos/result.tsx correctly 1`] = `
<div
  style="background: rgb(250, 250, 250); padding: 24px;"
>
  <div
    style="width: 100%;"
  >
    <div
      style="margin-bottom: 16px;"
    >
      <div
        class="ant-skeleton"
      >
        <div
          class="ant-skeleton-content"
        >
          <h3
            class="ant-skeleton-title"
            style="width: 185px;"
          />
        </div>
      </div>
      <div
        class="ant-skeleton ant-skeleton-element ant-skeleton-active"
      >
        <span
          class="ant-skeleton-button ant-skeleton-button-sm"
        />
      </div>
    </div>
    <div
      class="ant-card ant-card-bordered"
    >
      <div
        class="ant-card-body"
      >
        <div
          style="display: flex; justify-content: center; align-items: center; flex-direction: column; padding: 128px;"
        >
          <div
            class="ant-skeleton ant-skeleton-element"
          >
            <span
              class="ant-skeleton-avatar ant-skeleton-avatar-circle"
              style="width: 64px; height: 64px; line-height: 64px; margin-bottom: 32px;"
            />
          </div>
          <div
            class="ant-skeleton ant-skeleton-element ant-skeleton-active"
          >
            <span
              class="ant-skeleton-button"
              style="width: 214px; margin-bottom: 8px;"
            />
          </div>
          <div
            class="ant-skeleton ant-skeleton-element ant-skeleton-active"
          >
            <span
              class="ant-skeleton-button ant-skeleton-button-sm"
              style="width: 328px;"
            />
          </div>
          <div
            class="ant-space ant-space-horizontal ant-space-align-center"
            style="margin-top: 24px;"
          >
            <div
              class="ant-space-item"
              style="margin-right: 8px;"
            >
              <div
                class="ant-skeleton ant-skeleton-element ant-skeleton-active"
              >
                <span
                  class="ant-skeleton-button"
                  style="width: 116px;"
                />
              </div>
            </div>
            <div
              class="ant-space-item"
            >
              <div
                class="ant-skeleton ant-skeleton-element ant-skeleton-active"
              >
                <span
                  class="ant-skeleton-button"
                  style="width: 116px;"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
